startup.m

```diff
@@ -0,0 +1,6 @@
+% startup.m
+setPath ;
+setenv('DJ_HOST', 'at-database.ad.bcm.edu');
+setenv('DJ_USER','spatel');
+setenv('DJ_PASS','spatel');
```

Comment:
it is probably worth doing a squash of these two commits together and then doing a force push to your repository to overwrite them, so when the code is merged that never shows up in the history